docs/architecture/whitelisting.md

```diff
@@ -75,3 +76,60 @@ the following procedure:
 After these steps are done, the ERC20 whitelisting is finished. Liquidity for
 the given asset should promptly begin deposits in order to provide a smooth
 experience for users interested in interacting with the token.
+
+## Detailed Procedure
+### Prerequisites for Submission
+
+Before initiating a governance proposal to whitelist an ERC-20 token as a ZRC-20, ensure you meet the following prerequisites:
+
+* **Zetacored CLI Installation**: Make sure the zetacored CLI tool is installed on your system. This tool is essential for interacting with the Zetachain network and submitting proposals.
+* **Wallet Setup and Funding**: A wallet must be set up and associated with the zetacored CLI. This wallet should contain a minimum of 1000 ZETA tokens, plus additional ZETA for gas fees, to cover the deposit required for proposal submission.
+
+### **Submitting a Governance Proposal**
+
+To submit a governance proposal for whitelisting an ERC-20 token, follow these steps:
+
+1.  **Prepare the Proposal JSON File**: Draft a `mainnet_proposal.json` file that outlines the details of the token you wish to whitelist. This file must include:
+    
+    -   The proposal title and description, clearly articulating the value and details of the token and how it benefits the Zetachain ecosystem.
+    -   A deposit amount in ZETA to accompany the proposal. Mainnet minimum is currently `1000 Zeta` or `1000000000000000000000azeta`
```

Comment:
I'd prefer not to hard-code the value (it might change at any point in time). If you want to expand on the deposit amount, please, add to this section in the gov docs:

https://github.com/zeta-chain/docs/blob/6c9c7aba02963825b332fb76f6ba6a9b076797b2/docs/developers/cli/governance/text.md?plain=1#L50

Reply:
I think its important to document current time of writing minimums as they are not listed anywhere in the docs and will confuse users.

Reply:
Not only are the listed, but there is a table that dynamically fetches the values from the chain:

<img width="1840" alt="Screenshot 2024-02-20 at 09 27 40" src="https://github.com/zeta-chain/docs/assets/332151/df04fa6c-ef5e-4d28-903d-1cfab7cbd745">

Reply:
Modified to link to overview page instead